File: lalxml/swig/octave/check-swiglalxml.m
## check SWIG Octave module wrapping the LALXML library
## Author: Karl Wette, 2011

function msg(str)
  disp([program_name, ": ", str]);
endfunction

## check module load
swiglalxml;
msg("passed module load");

## passed all tests!
msg("================");
msg("PASSED all tests");
msg("================");

File: lalxml/swig/python/check-swiglalxml.py
# check SWIG Python module wrapping the LALXML library
# Author: Karl Wette, 2011

import os

def msg(str):
    print(os.path.basename(__file__) + ": " + str)

# check module load
import swiglalxml
from swiglalxml import *
msg("passed module load")

# passed all tests!
msg("================")
msg("PASSED all tests")
msg("================")
